### PR TITLE
Feature/answersandquestiontest

### PR DESCRIPTION
**Pull request by Billwilke42**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- No answers or question component unit tests.

## What is the new behavior?

- Adds unit test for answers and questions component

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/Answers/Answers.test.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import '@testing-library/jest-dom/extend-expect';
+import Answers from './Answers';
+import '@testing-library/jest-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux'
+
+const store = createStore(() => ({playerName: 'test', questions: [[
```
Comment (lilianaweimer) [inline src/Answers/Answers.test.js:10]: Nice mock store!

File: src/Answers/Answers.test.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  "difficulty": "easy",
+  "question": "What is the capital of Indonesia?",
+  "correct_answer": "Jakarta",
+  "incorrect_answers": [
+  "Bandung",
+  "Medan",
+  "Palembang"
+  ]
+  }
+
```
Comment (KyleWong2510) [inline src/Answers/Answers.test.js:53]: Nice job with all that initial setup

File: src/Answers/Answers.test.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    const answer2 = getByText('Bandung')
+    const answer3 = getByText('Medan')
+    const answer4 = getByText('Palembang')
+
+    expect(answer1).toBeInTheDocument()
+    expect(answer2).toBeInTheDocument()
+    expect(answer3).toBeInTheDocument()
+    expect(answer4).toBeInTheDocument()
+  })
+})
```
Comment (KyleWong2510) [inline src/Answers/Answers.test.js:76]: Clean and easy to read!

File: src/Question/Question.test.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+          <Question  
+            question={question} 
+          />
+        </Provider>
+      </BrowserRouter>
+    )
+    const questionOnDOM = getByText('What is the capital of Indonesia?')
+
+    expect(questionOnDOM).toBeInTheDocument()
+  })
```
Comment (KyleWong2510) [inline src/Question/Question.test.js:56]: Good work on the unit tests!

**Pull request merged**